src/shared/components/CreditCardForm/index.tsx

```diff
@@ -1,20 +1,32 @@
-import React, {FC, useContext, useEffect, useState} from 'react'
+import React, {FC, useEffect, useRef, useState} from 'react'
 
 // Context
-import {CheckoutContext} from 'src/checkout/context/checkout'
+import {CreditCardParams} from 'src/client/unityRoutes'
+import {ZuoraClient} from 'src/types/billing'
 
 export const ZUORA_SCRIPT_URL =
   'https://apisandboxstatic.zuora.com/Resources/libs/hosted/1.3.0/zuora-min.js'
 export const ZUORA_ID = 'zuora_payment'
 
-const ZuoraPaymentForm: FC = () => {
-  // FIXME: Add onFocus functionality
-  const {zuoraParams, handleSubmit, inputs} = useContext(CheckoutContext)
+export interface Props {
+  zuoraParams: CreditCardParams
+  onSubmit?: (paymentMethodId) => void
+  zuoraClient?: ZuoraClient
+}
 
-  const [client, setClient] = useState(window.Z)
+// FIXME: Add onFocus functionality
```

Comment:
Let's get this in as the next PR since the cart abandonment work should be setup for it

Reply:
sounds good!